app/main.py

```diff
@@ -1 +1,42 @@
-#  write your code here
+class Animal:
+    hungry_animal = []
+
+    def __init__(self,
+                 name: str,
+                 appetite: int,
+                 is_hungry: bool = True) -> None:
```

Comment:
Place each argument including `self` on a new line so that there would be no arguments on the same line with opening and closing parenthesis. It will help to improve the code style and readability